ui-core/src/types/forms.ts

```diff
@@ -111,6 +111,19 @@ export type FormContentPage = BaseElement & {
   elements: FormElement[]
 }
 
+/**
+ *  Certain SurveyJS elements can take on multiple forms. For example, the "title" field
+ *  for a question could either be a string, or an object mapping language codes to strings.
+ *  "default" is always present; the other languages are arbitrary.
+ */
+export type I18nMapElement = {
```

Comment:
I don't like this name, but I haven't yet decided what I'm going to rename it to

Reply:
I would just call this `I18nMap`